src/VirtoCommerce.QuoteModule.Data/Services/QuoteRequestService.cs

```diff
@@ -140,7 +140,9 @@ public async Task<QuoteRequestSearchResult> SearchAsync(QuoteRequestSearchCriter
                     }
                     else if (criteria.Keyword != null)
                     {
-                        query = query.Where(x => x.Number.Contains(criteria.Keyword));
+                        query = query.Where(x => x.Number.Contains(criteria.Keyword)
```

Comment:
?Could you please check that `CustomerName` and `Status` are not null to prevent excception?

Reply:
Added null checks in my commit